students/jsward/session02/grid_printer.py

```diff
@@ -0,0 +1,42 @@
+
+def print_grid(grid_size, grid_qty):
```

Comment:
I believe this is just a copy of your original print_grid2. You can just use the other function for the initial, the only thing you would want to do is hardcode the cellsize to be 2. One way to handle that is to make the default value 2:
def print_grid2(grid_size, grid_qty=2):

Nice job, overall!